frontend/src/main/web/src/components/Icons/index.jsx

```diff
@@ -0,0 +1,16 @@
1.17-2.342zm-8.163-11.697v3.5c0 1.287-.454 2.388-1.367 3.3S19.288 19 18 19s-2.387-.454-3.3-1.367-1.367-2.012-1.367-3.3v-3.5c0-1.287.454-2.387 1.367-3.3s2.012-1.367 3.3-1.367 2.387.454 3.3 1.367 1.367 2.013 1.367 3.3zM9.375 7c0-1.287.454-2.388 1.367-3.3s2.013-1.367 3.3-1.367 2.387.454 3.3 1.367c.038.038.075.104.113.154a6.817 6.817 0 0 0-2.967.917c-1.07.626-1.92 1.476-2.546 2.547S11 9.563 11 10.834v3.17c-.083-.066-.188-.132-.258-.207-.912-.908-1.367-2.008-1.367-3.296V7zm-5.35 17.5h-.483c-.32 0-.596-.113-.825-.342s-.342-.504-.342-.825V21.03c0-.08.05-.184.146-.314s.243-.283.434-.458c.192-.175.47-.375.837-.592s.784-.425 1.255-.62c1.167-.49 2.533-.864 4.104-1.134a28.8 28.8 0 0 1 2.65-.32c.35.662.79 1.253 1.328 1.778-1.775.275-3.35.683-4.725 1.225s-2.454 1.188-3.237 1.93c-.662.624-1.042 1.287-1.142 1.974zM4 24.863c0-.042 0-.083.004-.125C4 24.776 4 24.82 4 24.863zm.008-.18c0-.02.004-.042.004-.063-.004.02-.004.042-.004.063zm25.66 2.484c0 .32-.114.596-.343.825s-.504.342-.825.342h-21c-.32 0-.596-.113-.825-.342s-.342-.504-.342-.825v-2.304c0-.08.05-.183.146-.313s.24-.283.432-.458c.192-.175.47-.375.838-.592s.783-.425 1.254-.62c1.167-.49 2.533-.864 4.104-1.134s3.204-.413 4.892-.413 3.32.133 4.892.404c1.57.27 2.938.65 4.104 1.133.467.2.887.408 1.254.63s.646.416.837.59c.192.176.337.33.433.46s.146.237.146.312v2.304z"/></symbol><symbol viewBox="0 0 32 32" id="Icon-version"><path d="M0 9.333l16-8 16 8-16 8zm16 16.344l13-6.51 3 1.5-16 8.01-16-8.01 3-1.5zm0-5.667l13-6.5 3 1.5-16 8-16-8 3-1.5zM26.042 9.333L16 4.313 5.958 9.332 16 14.353z"/></symbol><symbol viewBox="0 0 32 32" id="Icon-warning"><path d="M14.003 2.154l-12 23.692h24l-12-23.692zm1.756-1.032l1.874 3.702 4.125 8.145c2.25 4.44 2.25 4.44 4.124 8.143l1.875 3.702C28.485 26.25 27.52 28 26.002 28h-24C.485 28-.48 26.25.247 24.815l12-23.693c.758-1.496 2.754-1.496 3.512 0z"/><circle cx="14" cy="22" r="2"/><path d="M12 11s0-2 2-2 2 2 2 2l-.875 7.003c-.07.55-.57.997-1.125.997-.552 0-1.057-.453-1.125-.997L12 11z"/></symbol><symbol viewBox="0 0 32 32" id="Icon-zanata"><defs><path id="cpa" d="M0 13.736c0 7.553 6.122 13.697 13.653 13.697 7.53 0 13.653-6.144 13.653-13.697C27.306 6.183 21.184.04 13.653.04 6.123.04 0 6.182 0 13.735z"/></defs><path d="M16.27 2.296C8.74 2.296 2.62 8.44 2.62 15.99c0 7.552 6.12 13.697 13.65 13.697s13.65-6.145 13.65-13.696c0-7.55-6.12-13.694-13.65-13.694zm0 2.347c6.286 0 11.366 5.095 11.366 11.408S22.556 27.46 16.27 27.46 4.904 22.363 4.904 16.05 9.984 4.644 16.27 4.644z"/><g transform="translate(2.617 2.196)"><mask id="cpb"><use xlink:href="#cpa"/></mask><path d="M22.017 12.187s-2.697 3.31-5.315 7.9l10.134 10.636s-9.505-4.53-19.366-10.1c1.765-3.062 4.113-6.16 4.113-6.16S6.79 12.51.27 9.058C3.1 4.79 6.585 1.256 6.585 1.256s7.18 5.48 15.433 10.93z" opacity=".1" mask="url(#cpb)"/></g><path d="M18.44 19.903l10.134 10.635s-9.505-4.53-19.366-10.1c1.765-3.062 4.113-6.16 4.113-6.16S8.53 12.324 2.01 8.874C4.84 4.605 8.32 1.072 8.32 1.072s7.18 5.48 15.433 10.93c0 0-2.697 3.31-5.315 7.9z"/></symbol></svg>'}
```

Comment:
Is this the entire icons.svg file inlined in here?
If so, was there an attempt to have webpack insert it from the file instead? If we can do it we should to avoid having duplicate data, and possible wasted time updating the original and wondering why it does not have an effect.